### ISSUE TITLE
Displaying several categories on one page

### ISSUE THREAD

**Issue by claudep** [bitmazk/django-calendarium#26]
In our use case, we'd like to show 2 different categories of events on the same page. If I read the code well, we can currently only choose between one or all categories, right?

**Comment by Tyrdall** [comment]
Yes. We only have a 'current category' https://github.com/bitmazk/django-calendarium/blob/master/calendarium/views.py#L53
